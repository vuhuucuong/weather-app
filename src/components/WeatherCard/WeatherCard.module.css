.dayWeatherCard {
  background: linear-gradient(to right, #7474bf, #348ac7);
  background-position: center;
  background-size: cover;
  border-radius: 10px;
  padding: 1rem;
  height: 500px;
}

.currentTemp {
  line-height: 1;
}
.weatherIcon {
  width: 70px;
}
