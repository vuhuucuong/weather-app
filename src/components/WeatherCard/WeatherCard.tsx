import React from 'react'
import cx from 'classnames'
import styles from './WeatherCard.module.css'

interface WeatherCardProps {}

const WeatherCard: React.FC<WeatherCardProps> = ({ children }) => {
  return (
    <div
      className={cx(
        'w-100 d-flex flex-column justify-content-center align-items-center text-light border-0 shadow-sm overflow-auto',
        styles.dayWeatherCard
      )}
    >
      {children}
    </div>
  )
}

export default WeatherCard
