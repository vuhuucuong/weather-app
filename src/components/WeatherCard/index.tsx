/* istanbul ignore file */
export { default } from './WeatherCard'
