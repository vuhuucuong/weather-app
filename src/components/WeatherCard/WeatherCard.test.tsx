import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import WeatherCard from './WeatherCard'

it('should match snapshot', () => {
  const tree = renderer
    .create(
      <WeatherCard>
        <div>hi</div>
      </WeatherCard>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders without crashing', () => {
  render(
    <WeatherCard>
      <div>hi</div>
    </WeatherCard>
  )
})
