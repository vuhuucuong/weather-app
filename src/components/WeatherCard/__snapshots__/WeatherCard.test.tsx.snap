// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="w-100 d-flex flex-column justify-content-center align-items-center text-light border-0 shadow-sm overflow-auto dayWeatherCard"
>
  <div>
    hi
  </div>
</div>
`;
